### ISSUE TITLE
Showing commit messages

### ISSUE THREAD

**Issue by akaBaum** [ls1intum/Artemis#7001]
### Describe the solution you'd like

Show the commit message in Artemis, where the Test results are shown.

It would help with identifying the commits, when they give different results.
It would also be an incentive to write somewhat decent commit messages. 

### Describe alternatives you've considered

Guessing which commit is which, depending on the commit times, how many commits have happened since.
Opening Bit-Bucket.

**Comment by krusche** [comment]
Good idea. @tobias-lippert is currently implementing a commit history view for instructors in the context of exams. 

@tobias-lippert what is the status?

We could then also use this information in other places such as in the history for students.

We might want to save the commit message as part of the programming submission in the database. This would be faster than retrieving it from git each time.
